static/app/views/issueDetails/groupTags/tagDistribution.tsx

```diff
@@ -30,6 +25,78 @@ export function TagDistribution({tag, groupId}: {groupId: string; tag: GroupTag}
   const otherDisplayPercentage =
     otherPercentage < 1 ? '<1%' : `${otherPercentage.toFixed(0)}%`;
 
+  return (
+    <TagPanel>
+      <TagHeader>
+        <Tooltip title={tag.key} showOnlyOnOverflow skipWrapper>
+          <TagTitle>{tag.key}</TagTitle>
+        </Tooltip>
+      </TagHeader>
+      <TagValueContent>
+        {visibleTagValues.map((tagValue, tagValueIdx) => {
+          const percentage = Math.floor(percent(tagValue.count, tag.totalValues));
+          const displayPercentage = percentage < 1 ? '<1%' : `${percentage.toFixed(0)}%`;
+          return (
+            <TagValueRow key={tagValueIdx}>
+              <Tooltip delay={300} title={tagValue.name} skipWrapper>
+                <TagValue>
+                  {tag.key === 'release' ? (
+                    <Version version={tagValue.name} anchor={false} />
+                  ) : (
+                    <DeviceName value={tagValue.name} />
+                  )}
+                </TagValue>
+              </Tooltip>
+              <Tooltip
+                title={tct('[count] of [total] tagged events', {
+                  count: tagValue.count.toLocaleString(),
+                  total: tag.totalValues.toLocaleString(),
+                })}
+                skipWrapper
+              >
+                <TooltipContainer>
+                  <TagBarValue>{displayPercentage}</TagBarValue>
+                  <TagBar percentage={percentage} />
+                </TooltipContainer>
+              </Tooltip>
+            </TagValueRow>
+          );
+        })}
+        {hasOther && (
+          <TagValueRow>
+            <TagValue>{t('Other')}</TagValue>
+            <Tooltip
+              title={tct('[count] of [total] tagged events', {
+                count: (tag.totalValues - totalVisible).toLocaleString(),
+                total: tag.totalValues.toLocaleString(),
+              })}
+              skipWrapper
+            >
+              <TooltipContainer>
+                <TagBarValue>{otherDisplayPercentage}</TagBarValue>
+                <TagBar percentage={otherPercentage} />
+              </TooltipContainer>
+            </Tooltip>
+          </TagValueRow>
+        )}
+      </TagValueContent>
+    </TagPanel>
+  );
+}
+
+export function TagDistributionWithDetailsLink({
+  tag,
+  groupId,
+}: {
+  groupId: string;
+  tag: GroupTag;
+}) {
```

Comment:
I'd rename to like `TagDetailsLink` and make this take `children: ReactNode` as a prop and then replace `<TagDistribution tag={tag} />` with `{children}` down below.

At the callsite it would be like:
```
              {displayTags.map((tag, tagIdx) => (
                <TagDetailsLink
                  tag={tag}
                  key={tagIdx}
                  groupId={group.id}>
                  <TagDistribution tag={tag} />
                </TagDetailsLink>
              ))}
```